cloudant-client/src/test/java/com/cloudant/tests/extensions/MultiExtension.java

```diff
@@ -0,0 +1,75 @@
+package com.cloudant.tests.extensions;
+
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.AfterTestExecutionCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.BeforeTestExecutionCallback;
+import org.junit.jupiter.api.extension.Extension;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+public class MultiExtension implements Extension, AfterAllCallback, AfterEachCallback,
```

Comment:
Remind me why this is necessary, was it that `@RegisterExtension` doesn't currently guarantee an order and `@ExtendWith` doesn't allow arguments to be passed?

Reply:
Yes, see https://github.com/junit-team/junit5/issues/506#issuecomment-368789516